### ISSUE TITLE
Blog project

### ISSUE THREAD

**Issue by Bartek532** [ajayns/react-projects#126]
I think blog project made with Next.js (SSG) may be very good project to include in this repository 😁

**Comment by HellBringer419** [comment]
Hey Nice idea ... I have some experience with Node also. I think we should start. Lemme comment after a simple setup and a db schema

**Comment by HellBringer419** [comment]
Started some work after forking this repo. Will push it here after work is complete. 
Checkout it out here: https://github.com/HellBringer419/react-projects/tree/blog_project_branch

Please check the blog/ and tell me if I should add you as a contributor.

**Comment by Bartek532** [comment]
Yeah, looks good, I think we should also add Headless CMS, where user can manage his blog posts :)

**Comment by Bartek532** [comment]
@HellBringer419 what do you think about using Wordpress as Headless CMS in this project? 😁

**Comment by HellBringer419** [comment]
Why do use WordPress? I believe you got my invite for collab too.
Sorry, I was  a bit busy. 

Also, what's done right now is not the project itself. I just initialized it. Work remains

**Comment by Bartek532** [comment]
Ok, I will look on it, but I think that WordPress is the most popular and if we use it as Headless CMS we should get neat performance ;)

**Comment by Bartek532** [comment]
Hmm, do you create any template or basic structure of the project? I don't see anything :)

**Comment by HellBringer419** [comment]
Yeah there is a folder 'blog' inside the 'blog_project_branch' branch
Link to the exact place: https://github.com/HellBringer419/react-projects/tree/blog_project_branch/blog

Seems like you are new to git.
Here's the how-to:
1. Clone my fork locally with ` $ git clone https://github.com/HellBringer419/react-projects.git `
2. Switch to the branch with ` $ git checkout blog_project_branch `
3. Move to blog folder ( ` $ cd blog ` )

You can then run it with yarn .... that's just the template btw. If you do any changes remember to pull then push. Also pull periodically to get latest changes. 
Also, you'll find a INFO.md file inside and I wanted to update it with the steps and knowledge so someone else can create their own thing with this (and hopefully learn). May change INFO.md to README.md later

**Comment by Bartek532** [comment]
Heh, ok, I didn't think about taking a look at another branch 😅, I already know everything, thanks, I'll be working on it 😁

**Comment by Bartek532** [comment]
Which CMS do you think to use?

**Comment by HellBringer419** [comment]
Didn't actually think through it. If it's actually possible, I need to learn this. 
Let's go with WordPress.

Also I added 2 funny blogs (static files for now) ;-p

**Comment by Bartek532** [comment]
Ok, but for WP you have to upload its files on the server, so you should have hosting or something like this.

**Comment by HellBringer419** [comment]
I think its an overhead then. Wouldn't it be a better idea to make the functionality with Next and then host it (like on heroku)?

The problem I referred to was integration. Like if I send over my files to WP server, would React play well with php. I mean if someone needs a WP site, why should they choose React (or Next) over php. Especially when everything is already built for the job.

Also, for the original question. Yeah I'm okay with sending over my js files to WP server provided it's the hosting is free. (WP has that offer btw)

**Comment by Bartek532** [comment]
Yeah, but I mean to use WordPress as Headless CMS, then fetch data on frontend (Next.js) and host on for example Vercel.

**Comment by Bartek532** [comment]
But ok, if you want use something else I agree ;)

**Comment by HellBringer419** [comment]
Now You got me interested. Let's do it your way. Use WP as headless CMS.

**Comment by Bartek532** [comment]
Ok, but do you provide WP on server?

**Comment by HellBringer419** [comment]
Not yet. Can't we like do the conf and then host it somewhere and then access it? Like an API. 
Please tell me if am missing something.

**Comment by Bartek532** [comment]
We should upload wp files to the server, then install wpgraphql plugin and on frontend with a link to our api fetch posts, comments etc.
